src/dash/student_view/Profile.jsx

```diff
@@ -1,48 +1,28 @@
 import React from 'react';
 import { connect } from 'react-redux';
```

Comment:
Be careful editting anything on profile why @SammyBaek is working on it as well

Reply:
yup!

Reply:
I believe he is working off this branch now so we should be good. I took all his changes this morning so there should be no conflicts if he didn't work on it today